sdxl-lora-swapping/model/model.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any
+
+from diffusers import DiffusionPipeline, AutoencoderKL
+import torch
+import base64
+from io import BytesIO
+from PIL import Image
+import time
+import functools
+
+
+# Good notebook to learn how to use diffusers + LoRA:
+# https://colab.research.google.com/gist/sayakpaul/109b7e792c64514fb3c057ecff4145ff/scratchpad.ipynb#scrollTo=GreOMxAcvlm8
+
+class Model:
+    def __init__(self, **kwargs) -> None:
+        self.pipe = None
+
+    def load(self):
```

Comment:
In cases where the kind and number of adapters + their size is small enough to fit in CPU RAM, is it still more performant to load these from remote to gpu mem rather than loading all in `load()` and on/offloading upon requests?

Reply:
if you have adapters, and you know them ahead of time, you should just use `hf_cache` or pop them on at a time in the load if you're ok with downloading the adapters.